docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
@defektive, ```image: metasploit``` works for me, especially with binstubs in place for easy execution of more that ```msfconsole```.